pkg/manifests/assets/dns/networkpolicy-allow.yaml

```diff
@@ -0,0 +1,29 @@
+# DNS Pods
+#  Egress to api server and upstream dns (can be wildcarded, so allow any egress)
+#  Ingress to dns on port 5353 (TCP and UDP) and to metrics (9154)
+apiVersion: networking.k8s.io/v1
+kind: NetworkPolicy
+# name, namespace,labels and annotations are set at runtime
+spec:
+  podSelector:
+    # matchLabels are set at runtime
+    matchLabels: {}
+  ingress:
+  - ports:
+    - protocol: TCP
+      port: 9154
+    - protocol: UDP
+      port: 5353
+    - protocol: TCP
+      port: 5353
+    - protocol: TCP
+      port: 8080
+    - protocol: TCP
+      port: 8181
```

Comment:
IIRC the health check ports don't need to be open, as kubelet should be able to reach the Pods with or without the network policy (unless ovn-kubernetes does differently). 

It is worth testing tho

Reply:
I was getting test errors (which surprised me) until these were opened.  I wasn't sure if something else was hitting them to determine when it was live.

Reply:
hum, weird.

I have just applied the network policy manually, so the port 8080 was blocked from a pod on another namespace but I could still curl it from kubelet:

```
 oc debug node/ip-10-0-86-223.us-east-2.compute.internal -- curl -kv http://10.131.0.5:8080/health
....
OK
```

then trying from my pod:
```
kubectl exec -it nginx-5869d7778c-pl7fq -- curl -m 3 http://10.131.0.5:8080/health
curl: (28) Connection timed out after 3000 milliseconds
```